docs/docs/api/turn-by-turn/api-reference.md

```diff
@@ -41,10 +41,11 @@ To build a route, you need to specify two `break` locations. In addition, you ca
 | `display_lon` | Longitude of the map location in degrees. If provided the `lat` and `lon` parameters will be treated as the routing location and the `display_lat` and `display_lon` will be used to determine the side of street. Both `display_lat` and `display_lon` must be provided and valid to achieve the desired effect. |
-| `search_cutoff` | The cutoff at which we will assume the input is too far away from civilisation to be worth correlating to the nearest graph elements |
-| `node_snap_tolerance` | During edge correlation this is the tolerance used to determine whether or not to snap to the intersection rather than along the street, if the snap location is within this distance from the intersection the intersection is used instead. The default is 5 meters |
-| `street_side_tolerance` | If your input coordinate is less than this tolerance away from the edge centerline then we set your side of street to none otherwise your side of street will be left or right depending on direction of travel |
-| `street_side_max_distance` | The max distance in meters that the input coordinates or display ll can be from the edge centerline for them to be used for determining the side of street. Beyond this distance the side of street is set to none |
+| `search_cutoff` | The cutoff at which we will assume the input is too far away from civilisation to be worth correlating to the nearest graph elements. The default is 35 km. |
+| `node_snap_tolerance` | During edge correlation this is the tolerance used to determine whether or not to snap to the intersection rather than along the street, if the snap location is within this distance from the intersection the intersection is used instead. The default is 5 meters. |
+| `street_side_tolerance` | If your input coordinate is less than this tolerance away from the edge centerline then we set your side of street to none otherwise your side of street will be left or right depending on direction of travel. The default is 5 meters. |
+| `street_side_max_distance` | The max distance in meters that the input coordinates or display ll can be from the edge centerline for them to be used for determining the side of street. Beyond this distance the side of street is set to none. The default is 1000 meters. |
+| `street_side_cutoff` | Cutoff for which road classes the preferred side matters, when `preferred_side` is set to `same` or `opposite`. It should be a string containing one of the following values (from highest to lowest road-class): motorway, trunk, primary, secondary, tertiary, unclassified, residential, service_other. Ignores the preferred side for all road classes which are smaller than the given cutoff class. Default value is `service_other`, so the set preferred side matters for all road classes. |
```

Comment:
i find this difficult to read. i think for non-experts they will have a difficult time of it. maybe something like is marginally more clear:

```suggestion
| `street_side_cutoff` | Disables the `preferred_side` when set to `same` or `opposite` if the edge has a road class less than that provided by `street_side_cutoff`. The road class must be one of the following strings: motorway, trunk, primary, secondary, tertiary, unclassified, residential, service_other.  The default value is `service_other` so that `preferred_side` will not be disabled for any edges. |
```

i don't know its hard to explain something so deep in the code :smile: 

also we should deprecate the name `preferred_side` and start parsing `street_side` just so it matches the rest.. who the hell did that.. (a pr for another time of course)

Reply:
I like your description more. So lets use it.
I agree to the street_side name too. Changing afterwards is ofc always annoying since people are already using it already. But it should be changed.